x-pack/plugins/beats_management/types/kibana.d.ts

```diff
@@ -1,41 +0,0 @@
-/*
```

Comment:
This file got removed because it re-declares outdated types. This surfaces here because lens is importing from OSS kibana. The change is also going into master in a separate PR: https://github.com/elastic/kibana/pull/37624